modules/node_modules/@ncigdc/containers/FrequentMutationsChart.js

```diff
@@ -48,7 +48,7 @@ const FrequentMutationsChartComponent = compose(
 
   const bandWidth = (
     (width - chartMargin.right - chartMargin.left) /
-    (ssms.hits.edges.length + 1) / (showSurvivalPlot ? 2 : 1)
+    (ssms.hits.edges.length + 1) / (showSurvivalPlot ? 1 : 2)
```

Comment:
Ahh, I see it now. 
If thats the case, then it probably makes sense to just make the graph always full width, and then the parent can size the container.